### PR TITLE
Changes after review

### PR DESCRIPTION
**Pull request by Elm0xz**
I created pull request for you :)

### CODE REVIEW

File: src/game/Board.java [hunk 1]
```diff
@@ -48,32 +48,30 @@ public Snake snake() {
 		return snake;
 	}
 
-	public Food food() {
+	public FoodGenerator food() {
```
Comment (Elm0xz) [inline src/game/Board.java]: Here, food() method is actually a getter for FoodGenerator, so you should name this getFoodGenerator(). Same for snake().

File: src/game/Board.java [hunk 2]
```diff
@@ -48,32 +48,30 @@ public Snake snake() {
...
 
 		while (start) {
 
 			for (int i = 0; i < snake.getListSnakeSize(); i++) {
 
-				if (randomRect.intersects(snake.getRectFromSnakeList(i))) {
-					food.setRandomFoofPosition();
-					randomRect.setLocation(food.getRandomX(), food.getRandomY());
+				if (foodGenerator.generateFoodOnRandomPosition().intersects(snake.getRectFromSnakeList(i))) {
+					newFood = new Rectangle(foodGenerator.generateFoodOnRandomPosition());
```
Comment (Elm0xz) [inline src/game/Board.java:67]: Perfect :)

File: src/game/Snake.java [hunk 3]
```diff
@@ -28,25 +26,13 @@ public void addHeadRectList() {
...
-
-			x[i] = snakeStep;
-			y[i] = snakeStep;
-
-		}
+		Random randomStartPosition = new Random();
 
 		int randomX = randomStartPosition.nextInt(16);
-		int randomy = randomStartPosition.nextInt(16);
+		int randomY = randomStartPosition.nextInt(16);
```
Comment (Elm0xz) [inline src/game/Snake.java]: I see that numbers for random generator here are a bit different than in FoodGenerator. Does it work properly ingame?

File: src/game/Board.java (hunk 1 above)
Comment (pablor83) [reply src/game/Board.java]: I changed names this methods

File: src/game/Board.java (hunk 2 above)
Comment (pablor83) [reply src/game/Board.java:67]: Thanks :)

File: src/game/Snake.java (hunk 3 above)
Comment (pablor83) [reply src/game/Snake.java]: I changed these code lines a bit.

File: src/game/Board.java (hunk 1 above)
Comment (krzysztofradomski) [reply src/game/Board.java]: "I changed this method's name."

**Comment by krzysztofradomski** [comment]
Reject this pr if it is no longer valid, merge if it should be merged.

**Pull request merged**